doc/rados/operations/health-checks.rst

```diff
@@ -1157,6 +1157,26 @@ The location of an OSD can be found with::
 
   ceph osd find osd.<id>
 
+PENDING_CREATING_PGS
+____________________
+
+One or more OSD have pending PGs creation due to various reasons, like
+incorrect crush map, dead OSDs in crush map etc. Additional message
+"(PGs creation pending, temporarily increase 'osd_max_pg_per_osd_hard_ratio')"
+in the pending PGs creation health message denotes one or more OSD have withheld
+PGs creation due to overdose protection meaning the number of PGs on OSD
+is more or equal to `mon_max_pg_per_osd * osd_max_pg_per_osd_hard_ratio` value.
+
+In case PGs creation is withheld due to overdose protection, it is highly recommended
+to add more OSDs to the cluster to resolve this condition. A temporary solution is
+to set `osd_max_pg_per_osd_hard_ratio` to the value which will satisfy the
+condition of ``num_pgs < (osd_max_pg_per_osd_hard_ratio * mon_max_pg_per_osd)``
+will help to resolve the issue of pending PGs creation issue.
+The ``num_pgs`` for the OSD daemon can be queried with the
+following command, executed from the daemon's host::
+
+  ceph daemon osd.<id> status
+
```

Comment:
Suggest:

When PG creation is pending due to overdose protection, it is highly recommended
to add OSDs to the cluster.  This will resolve the problem by spreading PGs across more OSDs,
with each OSD holding fewer PGs, so long as the new capacity is added appropriately across CRUSH failure domains . A temporary solution is to increase `osd_max_pg_per_osd_hard_ratio` to a value that will satisfy the
condition of ``num_pgs < (osd_max_pg_per_osd_hard_ratio * mon_max_pg_per_osd)``.
The ``num_pgs`` count for a given OSD can be queried via the daemon's admin socket with the
following command, executed from the daemon's host::